docs/source/guides/user/chapters/configuring.rst

```diff
@@ -55,65 +55,24 @@ Avocado will read the config files present in the git repos, and will ignore
 the system wide config files. Running ``avocado config`` will let you know
 which files are actually being used.
 
-Plugin config files
--------------------
-
-There are two ways to extend settings of extra plugin configuration. Plugins
-can extend the list of files parsed by ``Settings`` object by using
-``avocado.plugins.settings`` entry-point (Python-way) or they can simply drop
-the individual config files into ``/etc/avocado/conf.d`` (linux/posix-way).
-
-`avocado.plugins.settings`
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-This entry-point uses ``avocado.core.plugin_interfaces.Settings``-like object
-to extend the list of parsed files. It only accepts individual files, but you
-can use something like ``glob.glob("*.conf")`` to add all config files inside a
-directory.
-
-You need to create the plugin (eg. ``my_plugin/settings.py``)::
-
-   from avocado.core.plugin_interfaces import Settings
-
-   class MyPluginSettings(Settings):
-       def adjust_settings_paths(self, paths):
-           paths.extend(glob.glob("/etc/my_plugin/conf.d/*.conf"))
-
-
-And register it in your ``setup.py`` entry-points::
-
-   from setuptools import setup
-   ...
-   setup(name="my-plugin",
-         entry_points={
-             'avocado.plugins.settings': [
-                 "my-plugin-settings = my_plugin.settings.MyPluginSettings",
-                 ],
-             ...
+Configuring via command-line
+----------------------------
 
-Which extends the list of files to be parsed by settings object. Note this
-has to be executed early in the code so try to keep the required deps
-minimal (for example the `avocado.core.settings.settings` is not yet
-available).
+Besides the configuration files, the most used options has also the hability to
```

Comment:
Suggestion: "Besides the configuration files, the most used features can also be configured by command-line arguments".  And in case you don't take the suggestion: s/hability/ability/.